DOCS-404, DOCS-405 and misc. changes

Lmariomini opened the pull request:
Updated to reflect UICIRC-228 and UICIRC-184 regarding deleting request and notice policies; clarified in-house-use text; After checking with Julie, deleted bullet point on how notices are sent; misc. wording changes.

Lmariomini committed:
DOCS-404, DOCS-405 and misc. changes

Updated to reflect UICIRC-228 and UICIRC-184 regarding deleting request and notice policies; clarified in-house-use text; misc. wording changes.

Lmariomini committed:
DOCS-404, DOCS-405 and misc. changes 

Updated to reflect UICIRC-228 and UICIRC-184 regarding deleting request and notice policies; clarified in-house-use text; misc. wording changes.

Lmariomini committed:
DOCS-404, DOCS-405 and misc. changes 

Updated to reflect UICIRC-228 and UICIRC-184 regarding deleting request and notice policies; clarified in-house-use text; after checking with Julie, deleted bullet point on how notices are delivered; misc. wording changes.

Lmariomini committed:
DOCS-404, DOCS-405 and misc. changes 

Updated to reflect UICIRC-228 and UICIRC-184 regarding deleting request and notice policies; clarified in-house-use text; after checking with Julie, deleted bullet point regarding sending of notices; misc. wording changes.

julianladisch commented on content/en/docs/Access/Additional topics/loans/loans.md:
```diff
@@ -6,6 +6,8 @@ description: >
 tags: ["subtopic"]
 ---
 
+<table style="border: 2px solid black"><tr><td><strong> This section of the documentation contains links to external sites. Please be advised that these sites are not maintained by the FOLIO Documentation Group and may not be aligned with the current release of FOLIO.</strong></td></tr></table>
```
`<table>` for non-tabluar data should be avoided if possible.

* For tabular data use the Markdown table syntax: https://github.github.com/gfm/#tables-extension-
* For non-tabular data use regular paragraphs.

Drawbacks of `<table>`:
* The inner text doesn't support Markdown.
* Other editors of the documentation may not understand HTML syntax.
* Requires effort to keep it accessible:
  * https://www.w3.org/WAI/WCAG22/Techniques/failures/F46.html
  * https://www.w3.org/WAI/WCAG22/Techniques/failures/F49.html
  * https://www.w3.org/WAI/WCAG22/Techniques/failures/F91.html

In this case a simple paragraph put into `**` is sufficient.

Lmariomini replied:
I agree with you, have made recommended changes.

julianladisch commented on content/en/docs/Access/Additional topics/loans/loans.md:
```diff
@@ -6,6 +6,8 @@ description: >
 tags: ["subtopic"]
 ---
 
+<table style="border: 2px solid black"><tr><td><strong> This section of the documentation contains links to external sites. Please be advised that these sites are not maintained by the FOLIO Documentation Group and may not be aligned with the current release of FOLIO.</strong></td></tr></table>
```
A reader may not consider `wiki.folio.org` an external site.

A reader may miss this warning at the beginning.

The easiest solution for the reader is putting this warning next to the affected link: "That site may be aligned with a different FOLIO release because it is not maintained by the FOLIO Documentation Group."

Lmariomini replied:
In the meeting of Nov. 8 it was decided it was best to put the warnings at the top of the section.

julianladisch commented on content/en/docs/Access/Additional topics/loans/loans.md:
```diff
@@ -6,6 +6,8 @@ description: >
 tags: ["subtopic"]
 ---
 
+<table style="border: 2px solid black"><tr><td><strong> This section of the documentation contains links to external sites. Please be advised that these sites are not maintained by the FOLIO Documentation Group and may not be aligned with the current release of FOLIO.</strong></td></tr></table>
```
"current" should be avoided because it is a moving target. When Quesnelia gets released it becomes the current version. What about "may be aligned with a different FOLIO release"?

Lmariomini replied:
Made recommended change.

Lmariomini committed:
DOCS-404, DOCS-405 and misc. changes 

Updated to reflect UICIRC-228 and UICIRC-184 regarding deleting request and notice policies; clarified in-house-use text; misc. wording changes; added agreed-upon external links warning text, with Julian's recommended suggestions.
